### PR TITLE
add note about keeping time in sync, and directions for using an apt mirror while installing elasticsearch.

### PR DESCRIPTION
**Pull request by julialongtin**
add note about keeping time in sync, and directions for using an apt mirror while installing elasticsearch.

## Checklist:

Please tick the following before making your PR:

* [X] I ran `make` on this branch to build the docs and there are **no WARNINGs**.
* [X] After running `make`, I looked at the output by opening build/html/index.html in a browser and I'm satisfied with the rendering.

### CODE REVIEW

**Comment by tiago-loureiro** [comment]
I think [this PR](https://github.com/wireapp/wire-server-deploy/pull/211) makes this obsolete?

**Comment by julialongtin** [comment]
nope! just most of it. re-labeled, and edited. ;)

File: src/how-to/install/ansible-VMs.rst [hunk 1]
```diff
@@ -17,6 +17,7 @@ Assumptions
 - A bare-metal setup (no cloud provider)
 - All machines run ubuntu 16.04 or ubuntu 18.04
 - All machines have static IP addresses
+- Time on all machines is being kept in sync
```
Comment (tiago-loureiro) [inline src/how-to/install/ansible-VMs.rst]: Small note that [here](https://github.com/wireapp/wire-docs/pull/30) we are checking only C* at the moment: but we should certainly check them all

File: src/how-to/install/ansible-dependencies.rst [hunk 2]
```diff
@@ -20,6 +20,8 @@ This assumes you're using python 2.7 (if you only have python3 available, you ma
    source $HOME/.poetry/env
    ln -s /usr/bin/python2.7 $HOME/.poetry/bin/python
 
+During the installation, answer 'Y' to allow the Path variable for this user to be modified.
```
Comment (tiago-loureiro) [inline src/how-to/install/ansible-dependencies.rst]: Can't we have a `-y` or something that does this automatically? I have no idea if possible :)

**Comment by tiago-loureiro** [comment]
Looks good, have a look at the comments please.

**Pull request merged**